lecture_note.tex

```diff
@@ -881,7 +881,7 @@ \subsection{Early Stopping}
 and the validation cost as
 \begin{align}
     \label{eq:val_c}
-    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D(\hat{y}, y).
+    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D_{\var}(\hat{y}, y).

```

Comment:
Yes, that'll be better! How about you modify this PR to change the line 879 first? I'll merge it. Later either you can make a PR for the definition of ``D``, but I can do it myself.